modules/gif/image_frames_loader.cpp

```diff
@@ -88,7 +88,7 @@ void ImageFramesLoader::cleanup() {
 
 // ImageFrames
 
-RES ResourceFormatLoaderImageFrames::load(const String &p_path, const String &p_original_path, Error *r_error) {
+RES ResourceFormatLoaderImageFrames::load(const String &p_path, const String &p_original_path, Error *r_error, bool p_no_subresource_cache) {
```

Comment:
Signature change introduced in: https://github.com/godotengine/godot/pull/62408 There is examples of the variable being unused in other parts of Godot. I don't think this function has any form of caching.